IDEAS/Buildings/Components/BaseClasses/ConvectiveHeatTransfer/CavityAirflow.mo

```diff
@@ -0,0 +1,105 @@
+within IDEAS.Buildings.Components.BaseClasses.ConvectiveHeatTransfer;
+model CavityAirflow "Model for air flow through a cavity"
+  parameter Boolean linearise = false
+    "=true, to linearise the relation between heat flow rate and temperature difference";
+
+  parameter Real CD=0.65 "Discharge coefficient";
```

Comment:
this parameter is very unclear to me.

Reply:
@damienpicard I stole this parameter from the IBPSA model. It's essentially a coefficient that relates the flow expansion/acceleration to a pressure drop. From my understanding the coefficient is 1 in the isentropic expansion case, but additional flow friction causes the coefficient to be lower in reality. As far as I know this is a common parameter, see e.g. https://en.wikipedia.org/wiki/Orifice_plate .